Formula/python.rb

```diff
@@ -68,9 +68,9 @@ class Python < Formula
     sha256 "c075353337f9ff3ccf8091693d278782fcdff62c113245d8de43c5c7acc57daf"
   end
 
-  # Patch for compiling universal binaries on macOS 10.12
+  # Patch for compiling universal binaries on macOS 10.12, as well on 10.11 with Xcode 8
   # https://bugs.python.org/issue27806
-  if MacOS.version >= :sierra && build.universal?
+  if (MacOS.version >= :sierra || MacOS::Xcode.installed? && MacOS::Xcode.version >= "8.0") && build.universal?
```

Comment:
Not everything is covered by RuboCop. We strive to stay within 80 columns (and I'm not the one that police this sort of thing the hardest), but in this case I think it's reasonable to just put build.universal? on the second line (I'm happy with 100 in a case like this, and others who police harder break the rule themselves too — see erlang.rb for instance — so it should be fine).